test/api_getProductRaw_test.dart

```diff
@@ -77,7 +77,7 @@ void main() {
       expect(
           result.product!.nutrientLevels!.levels[NutrientLevels.NUTRIENT_SALT],
           Level.MODERATE);
-          print(result.product!.countries);
+      print(result.product!.countries);
```

Comment:
Thanks @FredJul for the new changes, can you please delete the print message to avoid having messages during the tests running.

Reply:
that's a good remark :) it's done